### ISSUE TITLE
Repo for Isengard.

### ISSUE THREAD

**Issue by lostwithouthope** [pipelka/xbmc-addon-xvdr#155]
Yesterday I've tried the Openelec 6 beta. As they no longer include the pvr addons, you have to install them from their repository. Your xvdr addon is missing there. I don't know if this is intended or not.

So I wanted to install them from your repository (http://www.xvdr.tv) but there is no repo for isengard yet?

So I assume I have to build your addon atm by myself?

Thanks for your effort and work.

**Comment by pipelka** [comment]
Yes. ATM you have to build it on your own ("master" branch). I primarily
concentrate myself on the stable kodi branch.

Regards,
Alex


2015-05-11 8:58 GMT+02:00 lostwithouthope <notifications@github.com>:

**Comment by lostwithouthope** [comment]
Thanks for the info.

Cheers

**Comment by lostwithouthope** [comment]
I'll reopen this just as a hint that kodi isengard is now released.
Cheers.

**Comment by jzvc** [comment]
Yep, now for ubuntu, 14.2 removed from repos ??? and there only 13 or 15.
